### PR TITLE
refactor(cleanup): clean up AREnableConversationalBuyNow

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

 clean up AREnableConversationalBuyNow feature flag https://github.com/artsy/echo/pull/541

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- clean up AREnableConversationalBuyNow - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (clean up AREnableConversationalBuyNow - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1a87d007d6dcf620db3a016e9445e01ee9087649
</p>

File: src/app/store/config/features.ts [hunk 1]
```diff
@@ -56,10 +56,10 @@ export const features: { [key: string]: FeatureDescriptor } = {
     description: "Enable My Collection Price Estimate Range",
     showInDevMenu: false,
   },
-  AREnableConversationalBuyNow: {
+  AREnableArtworksFromNonArtsyArtists: {
```
Comment (MrSltun) [inline src/app/store/config/features.ts:59]: Are we replacing the feature? Or is this some weird diff? 👀

**Review by MrSltun — APPROVED** [review]
Looks great!! 🌟 Just added one question

File: src/app/store/config/features.ts (hunk 1 above)
Comment (gkartalis) [reply src/app/store/config/features.ts]: must have been from switching branches 🤑

**Pull request merged**